src/components/AdvancedOptions.tsx

```diff
@@ -1,3 +1,4 @@
+import "./AdvancedOptions.scss"
```

Comment:
No, moving away from css modules is the wrong direction. CSS modules namespace classnames to prevent conflicts from other files.

I recommend following Arthur's suggesting of using `data-testid` to help with element targeting

Reply:
Yup just add `data-testid` to components needing targeting and pass reasonable name through as the value to that attribute.